components/atoms/forms/FileInput/FileInput.js

```diff
@@ -53,14 +54,14 @@ export default function FileInput({
 
   const handleChange = (event) => {
     if (event.target.files.length > 0) {
-      updateFiles([...event.target.files], "add");
+      updateFiles(setFiles, [...event.target.files], "add");
```

Comment:
updateFiles and setFiles, both are passed as props and used together, so individually they have no use. The functionality of setFiles can be encompassed in the updateFiles function itself.